arelle/ModelDocument.py

```diff
@@ -327,7 +327,19 @@ def load(modelXbrl, uri, base=None, referringElement=None, isEntry=False, isDisc
         elif _type == Type.INSTANCE:
             modelDocument.instanceDiscover(rootNode)
         elif _type == Type.INLINEXBRL:
-            modelDocument.inlineXbrlDiscover(rootNode)
+            try:
+                modelDocument.inlineXbrlDiscover(rootNode)
+            except RecursionError as err:
```

Comment:
(Arbitrary line.)

Can you show an example stack trace of the error this is fixing?  I'm curious what the recursion actually is.

My Python is not strong, but it seems somewhat crude to catch a RecursionError rather than limiting whatever method is recursing.  Will this "swallow" the stack trace if we have some other unrelated infinite recursion?